sources/__tests__/getRole.js

```diff
@@ -89,7 +89,8 @@ const cases = [
 	["html", null, createElementFactory("html", {})],
 	["iframe", null, createElementFactory("iframe", {})],
 	["img with alt=\"some text\"", "img", createElementFactory("img", {alt: "text"})],
-	["img with alt=\"\"", null, createElementFactory("img", {alt: ""})],
+	["img with alt=\"\"", "presentation", createElementFactory("img", {alt: ""})],
+	["img no alt attribute", "presentation", createElementFactory("img", {})],
```

Comment:
omitting an `alt` attribute should not remove it from the AOM. setting it to empty string, or omitting the _value_ of the attribute _should_ remove the image. see https://github.com/eps1lon/dom-accessibility-api/issues/429#issuecomment-696240405

Reply:
They're not hidden from the AOM. They just have another role. See https://github.com/eps1lon/dom-accessibility-api/issues/429#issuecomment-696248935 for how to proceed.

Reply:
Should probably add a test for `<img aria-label="the image" />`. Maybe you're concerned about that use case?